gaia-button.js

```diff
@@ -24,6 +24,24 @@ module.exports = component.register('gaia-button', {
     this.setupShadowRoot();
     this.setAttribute('role', 'button');
     this.tabIndex = 0;
+    if (this.keyboardEnabled) {
```

Comment:
If there is no memory overhead I think we should just make them keyboard accessible by default. Otherwise TV apps will have to decorate every button with `keyboardEnabled`.